.github/workflows/integration-tests.yml

```diff
@@ -10,6 +10,42 @@ on:
         description: 'Commit hash to run the tests'
         required: true
 jobs:
+  build-quesma-docker-image:
+    strategy:
+      matrix:
+        module: [ "quesma" ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with: ## @TODO REMOVE
+          ref: ${{ github.event.inputs.GIT_REF }}
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          cache-dependency-path: ${{ matrix.module }}/go.sum
+          go-version: '1.22'
+
+      - name: Build and export
+        uses: docker/build-push-action@v6
+        with:
+          context: ${{ matrix.module }}/.
+          tags: ${{ matrix.module }}:latest
+          outputs: type=docker,dest=/tmp/image.tar
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+        env:
+          DOCKER_BUILD_SUMMARY: false
+
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ matrix.module }}
+          path: /tmp/image.tar
+          retention-days: 1
```

Comment:
Blatant copy-paste from another workflow. This copy-paste will get fixed by: https://github.com/QuesmaOrg/quesma/pull/862

Reply:
maybe let's merge #862 and apply it then ?

Reply:
👍

Reply:
Fixed in the latest version